tla/Traceccfraft.tla

```diff
@@ -348,6 +364,24 @@ TraceNextConstraint ==
               \* BP:: line below is the first step towards diagnosing a divergence. Once
               \* hit, advance evaluation with step over (F10) and step into (F11).
               BP::
+              /\ \A s \in (Servers \ {logline.msg.state.node_id}) :
+                 /\ state[s] = state'[s]
+                 /\ log[s] = log'[s]
+                 /\ state[s] = state'[s]
+                 /\ currentTerm[s] = currentTerm'[s]
+                 /\ nextIndex[s] = nextIndex'[s]
+                 /\ matchIndex[s] = matchIndex'[s]
+                 /\ commitIndex[s] = commitIndex'[s]
+                 /\ votedFor[s] = votedFor'[s]
+              \* json state logging in raft.h is inconsistent; sometimes it logs the state before
```

Comment:
Happy to fix that. What's best, before or after?

Reply:
Let's wait with this cleanup because I want to do a bigger refactoring after fancy_election.2 and suffix_collision.2 validate.